scripts/collie-cli.test.sh

```
@@ -1199,17 +1300,44 @@ assert_contains "$(cat "$U_CALLS")" "${MANAGED}\$ bun ${MANAGED}/cli/main.ts _ap
 git_q -C "$ORIGIN" commit -q --allow-empty -m "third"
+CLONE_BRANCH_AT="$(git -C "$CLONE" rev-parse HEAD)"
+health_says 9.10.0
 upd "$CLONE" "$BIN" update || fail "\`collie update\` failed on a linked clone: ${STDERR}"
-assert_contains "$STDOUT" "git pull --ff-only"
-assert_eq "$(git -C "$CLONE" rev-parse HEAD)" "$(git -C "$ORIGIN" rev-parse HEAD)"
+assert_contains "$STDOUT" "staged checkout"
+# The verb STAGES and hands off (M15/04): the swap, the restart and the health gate all run in a
+# process with its own lifetime, because the restart would otherwise kill the bridge that asked.
+assert_contains "$STDOUT" "handed off to systemd-run --user --collect"
+assert_contains "$STDOUT" "Watch it with: collie update --status"
+wait_for_run done
```

Shellcheck (0.11.0)

[warning] Use semicolon or linefeed before 'done' (or quote to make it literal).

(SC1010)